docs/getting-started/get-started-using-claudie.md

```diff
@@ -0,0 +1,15 @@
+# Get started using Claudie
```

Comment:
I don't really like that we have a duplicate of this in the README.
Is there something we can do about that?

Reply:
Yes that's correct, I'm just wondering if there's a way somehow to do it without "duplicate code"

If not, no worries

Reply:
I guess the only way, how to do this is through file inclusion in markdown.
IMO it is not worthed, because it will complicate the code way too much. Due to the fact, that we are copying just 2 last steps and last sentence from `Get started using Claudie` in our main README.md and if you look closely, the paths in the docs links aren't the same.

Reply:
ok